lib/kwtool/crawler/keywords.ex

```diff
@@ -16,9 +16,6 @@ defmodule Kwtool.Crawler.Keywords do
 
       {:error, :file_is_empty} ->
         {:error, :file_is_empty}
-
-      {:error, :file_is_invalid} ->
-        {:error, :file_is_invalid}
```

Comment:
We're removing this because it is already handled by the file validator, right? What about the `{:error, :file_is_empty}` case, maybe we can do the same 🤔 

I think I'm also ok with the previous implementation (keeping error tuples), It would also raise the exception in the controller, right? 💭

Reply:
I removed this section because, the `parse(_)` now raises expcetion instead of return the `{:error, :file_is_invalid}` tuple. So the return result of `UploadParser.parse(keyword_file)` no longer contains this error tuple.
- Further reading: https://keathley.io/blog/good-and-bad-elixir.html ("Raise exceptions if you receive invalid data")